tests/test_app/TestApp/Model/Table/CustomCommentsTable.php

```diff
@@ -0,0 +1,65 @@
+<?php
+declare(strict_types=1);
+/**
+ * CakePHP(tm) : Rapid Development Framework (https://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * Redistributions of files must retain the above copyright notice
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
+ * @since         4.0.0
+ * @license       https://opensource.org/licenses/mit-license.php MIT License
+ */
+namespace TestApp\Model\Table;
+
+use Cake\Datasource\EntityInterface;
+use Cake\ORM\Query;
+use Cake\ORM\Table;
+
+class CustomCommentsTable extends Table
+{
+    /**
+     * initialize hook
+     *
+     * @param array $config Config data.
+     * @return void
+     */
+    public function initialize(array $config): void
+    {
+        $this->setTable('comments');
+        $this->setAlias('Comments');
+    }
+
+    /**
+     * Finder using variadic args.
+     *
+     * @param Query $query Query
+     * @param bool $published Whether or not a comment is published.
+     * @return Query
+     */
+    public function findPublished(Query $query, bool $published): Query
```

Comment:
I probably wasn't clear in previous comment. By "2 args" i meant 2 additional args besides `$query` :slightly_smiling_face:.

Reply:
Working on it